packages/@aws-cdk/aws-lambda-nodejs/package.json

```diff
@@ -63,7 +63,7 @@
     "cdk-build-tools": "0.0.0",
     "cdk-integ-tools": "0.0.0",
     "delay": "4.4.0",
-    "parcel": "2.0.0-beta.1",
+    "parcel": "2.0.0-nightly.407",
```

Comment:
not sure this is a change we'd want to pick up. are the nightly builds reliable?
thoughts on whether we want to include/exclude this one? @jogold @RomainMuller

Reply:
Yes, we should exclude `parcel` (`--reject`) for the moment

Reply:
#10499 adds the exception